src/S3/Transfer.php

```diff
@@ -338,6 +341,7 @@ private function addDebugToBefore($debug)
                     $source = "s3://{$command['Bucket']}/{$command['Key']}";
                     $dest = $command['@http']['sink'];
                     break;
+                case 'CompleteMultipartUpload':
```

Comment:
Nah, all that's there is:

- PutObject
- UploadPart
- CreateMultipartUpload
- CompleteUploadPart

Should I just replace `CompleteUploadPart` with `CompleteMultipartUpload`?